docs/typelibrary/isobus/UT/io/Auxiliary/IN/Aux_IXA.md

```
@@ -0,0 +1,61 @@
+# Aux_IXA
+
+<img width="1383" height="318" alt="image" src="https://github.com/user-attachments/assets/d064bd8f-6af0-4b04-9952-995892584e9a" />
+
+* * * * * * * * * *
+
+## Einleitung
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)